migrations/20200322171627_tags.js

```diff
@@ -0,0 +1,53 @@
+exports.up = async knex => {
+  const buildTableSchema = [
+    {
+      tableName: "tag",
+      create: t => {
+        t.increments("id").primary();
+        t.text("group");
+        t.text("name").notNullable();
+        t.text("description").notNullable();
+        t.boolean("is_deleted").defaultTo(false);
+        t.timestamp("created_at")
+          .notNullable()
+          .defaultTo(knex.fn.now());
+        t.timestamp("updated_at")
+          .notNullable()
+          .defaultTo(knex.fn.now());
+        t.integer("organization_id").notNullable();
```

Comment:
Please create an index on organization_id (bonus for a single two-field index of [organization_id, is_deleted])

Reply:
Done! I haven't worked with indexes before. Worth a double check.